OSDOCS-2478: Adds procedure for enabling the multicluster console (#42063)
openshift/openshift-docs: modules/multi-cluster-about.adoc

// Module included in the following assemblies:
//
// * assemblies/web-console.adoc

:_content-type: CONCEPT
[id="multi-cluster-about_{context}"]
= Multicluster console

> The multicluster console provides a simplified hybrid cloud console that provides a single interface. You can switch between Advanced Cluster Managment (ACM) and the cluster Console in the same browser tab. It provides a consistent design that allows for shared components.

Reviewer: Or maybe: 
```suggestion
The multicluster console provides a single interface with consistent design for the hybrid cloud console. If you enable the feature, you can switch between Advanced Cluster Managment (ACM) and the cluster Console in the same browser tab. 
```

> The multicluster console provides a simplified hybrid cloud console that provides a single interface. You can switch between Advanced Cluster Managment (ACM) and the cluster Console in the same browser tab. It provides a consistent design that allows for shared components.

Author: I think I like the second suggestion best. Thanks I think it clears it up a bit more!
